doc/changelog/10-standard-library/10827-dedekind-reals.rst

```diff
@@ -0,0 +1,9 @@
+- New module `Reals.ClassicalDedekindReals` defines Dedekind real numbers
+  as boolean-values functions along with 3 logical axioms : limited principle
+  of omniscience and functional extensionality. The exposed type R in module
+  `Reals.Rdefinitions` is those Dedekind reals, hidden behind an opaque module.
+  Classical Dedekind reals are a quotient of constructive reals, which allows
+  to transport many constructive proofs to the classical case.
+
+  See `#10827 <https://github.com/coq/coq/pull/10827>`_, by Vincent Semeria,
+  with the help and review of Guillaume Melquiond and Bas Spitters.
```

Comment:
Are you anticipating on who is going to review your PR or is this a reflection of what happened at the WG? (I don't know since I wasn't there.)

Reply:
@vincentse: isn't there still something to clarify here?

Reply:
@herbelin Funext and Dedekind reals were first suggested by @spitters a few months ago. We then discussed it with @silene at the last working group in Nantes and he agreed.

Reply:
@VincentSe: Ah, ok, I see. Then there was apparently a misunderstanding on the word "review" which in the context of a PR tends to refer to "reviewed on GitHub" . So, a less ambiguous formulation could be "based on discussions with Bas Spitters and Guillaume Melquiond", or whatever similar fomulation reflecting well the situation. Sorry for the confusion.

Reply:
You could also credit Hugo for the review BTW :wink:

Reply:
You could also credit Hugo for the review BTW :wink:

Reply:
done